client/src/lib/prefixClassNames.js

```diff
@@ -0,0 +1,33 @@
+import classNames from 'classnames';
+
+/**
+ * Utility method wrap around classeNames to prefix the return class names
+ * @param cssPrefix
+ * @returns {Function}
+ */
+const prefixClassNames = (cssPrefix) => (...args) => {
```

Comment:
This is inspired by https://www.npmjs.com/package/classnames-prefix.

Felt a bit overkill to require an extra package for such a simple method. If we feel this is not the right place for this kind of method, I'm happy to remove it and hard code `element-editor__hover-bar` into the class names.

Reply:
Yeah I thought the same... I don't mind too much but I probably would've leaned towards writing them out before you'd added - now that it's there it can probably stay